types/trace/trace.go

```diff
@@ -37,7 +37,8 @@ type Event struct {
 	PodSandbox          bool         `json:"podSandbox"`
 	EventID             int          `json:"eventId,string"`
 	EventName           string       `json:"eventName"`
-	MatchedPolicies     uint64       `json:"matchedPolicies"`
+	MatchedPolicies     uint64       `json:"-"`                    // omit MatchedPolicies from json output
+	MatchedPolicesNames []string     `json:"matchedPoliciesNames"` // to be omited when issue #2868 is resolved
```

Comment:
```suggestion
	MatchedPolicesNames []string     `json:"matchedPoliciesNames"` // omit when nil after issue #2868 is resolved
```

Reply:
Should we omit this field when null? Thinking of CLI use case where there will be no policy.